de.tudarmstadt.ukp.wikipedia.api/src/main/java/de/tudarmstadt/ukp/wikipedia/api/Page.java

```diff
@@ -595,7 +595,7 @@ public String getPlainText()
 	 *         type of the go() method of your visitor.
 	 * @throws WikiApiException Thrown if errors occurred.
 	 */
-	public Object parsePage(AstVisitor v) throws WikiApiException
```

Comment:
Can you explain to me why the need to change the constructor here ?

Reply:
@tgalery This is not a constructor here. The method is private now, as this is _internal_ use only. No external caller should ever call `parsePage` from a consuming perspective.

Reply:
yeah sorry I wanted to say accessor, but good enough for me

Reply:
sure, np.